app-config.production.yaml

```diff
@@ -11,25 +11,19 @@ backend:
   baseUrl: http://localhost:7007
   # The listener can also be expressed as a single <host>:<port> string. In this case we bind to
   # all interfaces, the most permissive setting. The right value depends on your specific deployment.
-  listen: ':7007'
+  listen: ":7007"
 
   # config options: https://node-postgres.com/api/client
   database:
     client: pg
     connection:
-      host: ${POSTGRES_HOST}
-      port: ${POSTGRES_PORT}
-      user: ${POSTGRES_USER}
-      password: ${POSTGRES_PASSWORD}
+      host: ${host}
+      port: ${port}
+      user: ${user}
+      password: ${password}
```

Comment:
Any reasons to rename these variables?

Reply:
Yes the secret `janus-idp-pguser-janus-idp` that is created by the `PostgresCluster` contains values using these keys:
```yaml
kind: Secret
apiVersion: v1
metadata:
  name: janus-idp-pguser-janus-idp
  namespace: janus-idp
...
data:
  host: 
  password: 
  port: 
  user: 
  verifier:
type: Opaque

```

Reply:
ok, so we can rename the secret instead. We will have to add this in the https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md and having only `host, port`... are too generic names.

Reply:
As far as I know we can't rename the keys in the secret since its auto-generated by the postgres cluster deployment. However if you know of a way to do this I all open since I also think the keys are too generic.

I'll update the documentation accordingly thought.